datasets/es/cnp_wanted/es_cnp_wanted.yml

```
@@ -30,7 +31,7 @@ ci_test: false
 assertions:
   min:
     schema_entities:
-      Person: 1
+      Person: 8
```

this feels a bit tight now they've bumped the number of people

```suggestion
      Person: 6
```